### ISSUE TITLE
convertFromRaw infinite loop when content has anchor tags

### ISSUE THREAD

**Issue by madshargreave** [brijeshb42/medium-draft#17]
Hi

First off, awesome library - saves A TON of work.

However I've run into a problem. I persisting data to my DB as described in the wiki using

`convertToRaw(this.state.editorState.getCurrentContent());`

and then initializing it using

`createEditorState(rawContent)`

This works fine, unless there is any anchor tags in the content, in which case `createEditorState`'s `convertFromRaw` enters an infinite loop

UPDATE:

Probably catching the error yields:

```
Unknown DraftEntity key.
    at invariant (eval at <anonymous> (http://localhost:3000/site.js:596:2), <anonymous>:38:15)
    at Object.get (eval at <anonymous> (http://localhost:3000/site.js:11838:2), <anonymous>:71:27)
    at eval (eval at <anonymous> (http://localhost:3000/site.js:12474:2), <anonymous>:53:30)
    at Array.forEach (native)
    at convertFromDraftStateToRaw (eval at <anonymous> (http://localhost:3000/site.js:12474:2), <anonymous>:52:14)
    at Composer.handleChange (eval at 2247 (http://localhost:3000/1.bf94ea2aa48f3cb3c814.hot-update.js:6:2), <anonymous>:224:42)
    at Composer.proxiedMethod (eval at <anonymous> (http://localhost:3000/site.js:2241:2), <anonymous>:44:30)
    at MediumDraftEditor._this.onChange (eval at <anonymous> (http://localhost:3000/site.js:14256:2), <anonymous>:86:19)
    at MediumDraftEditor.setLink (eval at <anonymous> (http://localhost:3000/site.js:14256:2), <anonymous>:146:12)
    at Toolbar.onKeyDown (eval at <anonymous> (http://localhost:3000/site.js:13608:2), <anonymous>:136:20)
```

Any ideas?

**Comment by brijeshb42** [comment]
Working fine for me. Where are you getting the value of `rawContent` from ?

**Comment by madshargreave** [comment]
`rawContent` is the data form `convertToRaw()`

Tried the wiki, didn't work unfortunately

**Comment by brijeshb42** [comment]
Can you share few lines of the code? Not enough information for me to debug

**Comment by madshargreave** [comment]
Weird, updated from 0.3.10 to 0.3.12 and it seemed to fix the problems I was having

**Comment by brijeshb42** [comment]
It was because although the `Link` decorator was defined properly in `medium-draft`, it was not being applied to the editor. It was fixed in `0.3.12`

**Comment by madshargreave** [comment]
Got cha, much appreciated
